dask/tests/test_graph_manipulation.py

```diff
@@ -0,0 +1,300 @@
+import random
+import time
+from operator import add
+
+import pytest
+
+import dask
+from dask import delayed
+from dask.base import clone_key
+from dask.graph_manipulation import checkpoint, clone, bind, block_until_done, chunks
+from dask.highlevelgraph import HighLevelGraph
+from dask.tests.test_base import Tuple
```

Comment:
not terribly happy about importing a test class from another test module. I tried moving Tuple to dask.utils_test, but then I realized that that module is imported by a bunch of non-test modules (something that should be cleaned up in the future) and that caused a wealth of circular dependencies.